std_msgs/Header header

string type
geometry_msgs/Pose target

uint8 LEVEL_NORMAL        = 0
uint8 LEVEL_IMPORTANT     = 1
uint8 level
